### DISCUSSION TITLE
这几个月的连接稳定性太差了，下午和夜间直接进入无法连接状态，需要你们尽快解决一下，用了这么多年，用户开始不是用户了么？

### DISCUSSION THREAD

**Discussion by Sagefl009** [getlantern/lantern-forum-cn#1037] [General]
这几个月的连接稳定性太差了，下午和夜间直接处于无法连接状态，需要你们尽快解决一下，用了这么多年，用户开始不是用户了么？我写个反馈都要挑上午。

**Comment by MaxMKMax** [comment]
@Sagefl009 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)


当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

我们将非常感谢您。

**Comment by MaxMKMax** [comment]
@Sagefl009 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。
您也检查一下看您是否有下载最新版本. 谢谢您.
